src/services/shortLink/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Context, GraaspPlatform } from '@graasp/sdk';
+
+import { CLIENT_HOSTS, CLIENT_PREFIX } from '../../utils/config';
+
+export function getPlatformHost(shortLinkView: GraaspPlatform) {
+  const [BUILDER_HOST, PLAYER_HOST, LIBRARY_HOST] = CLIENT_HOSTS;
+
+  switch (shortLinkView) {
+    case GraaspPlatform.BUILDER:
+      return BUILDER_HOST;
+    case GraaspPlatform.LIBRARY:
+      return LIBRARY_HOST;
+    case GraaspPlatform.PLAYER:
+      return PLAYER_HOST;
+    default:
+      throw new Error('Invalid Shortlink view');
```

Comment:
I see that there is a disconnect between the views we allow in the types and the ones we provide here, not sure if we could write this differently (maybe with a map ?)

Reply:
Maybe see: https://github.com/graasp/graasp-ui/blob/main/src/PlatformSwitch/hooks.ts

Reply:
The new class (maybe Singleton) should solve this.